ci/docs/pulp_index_pulp3.rst

```diff
@@ -42,4 +41,4 @@ Index and Search
 .. toctree::
    :maxdepth: 1
 
-   all_content_index_pulp3
+   all_content_index
```

Comment:
Does it make sense or just more confusing? :)

I also suggest to postpone review of this PR just  a bit, while I am trying to make a proposal for building docs. And this particular piece of code may change.